src/include/rho/FixedVector.hpp

```diff
@@ -222,9 +222,8 @@ namespace rho {
 	// Virtual function of GCNode:
 	void detachReferents() override;
     private:
-	T* m_data;  // pointer to the vector's data block.
```

Comment:
Thanks.  We probably should revisit this in a year or so.  Once a bunch of other things have been improved, this might start to make a difference.